invenio_app_rdm/config.py

```diff
@@ -356,21 +358,9 @@ def files_rest_permission_factory(obj, action):
         "schedule": crontab(minute=0, hour=7),  # Every day at 07:00 UTC
     },
     # indexing of statistics events & aggregations
-    "stats-process-events": {
-        "task": "invenio_stats.tasks.process_events",
-        "schedule": timedelta(minutes=30),
-        "args": [("record-view", "file-download")],
-    },
-    "stats-aggregate-events": {
-        "task": "invenio_stats.tasks.aggregate_events",
-        "schedule": timedelta(hours=1),
-        "args": [
-            (
-                "record-view-agg",
-                "file-download-agg",
-            )
-        ],
-    },
+    "stats-process-events": StatsEventTask,
+    "stats-aggregate-events": StatsAggregationTask,
+    "reindex-stats": StatsRDMReindexTask,
```

Comment:
This is a new task, which means that the `app-rdm` could be release only after the stats and the rdm have been released. 

If you prefer, we could define the tasks here.